src/pages/graphql/payment-methods/payment-services.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Payment Services as payment method
+description: Learn how to use the GraphQL API mutation for the Payment Services solution.
+keywords:
+  - GraphQL
+  - Payments
+---
+
+# Payment services as payment method
+
+Payment Services for Adobe Commerce and Magento Open Source is your turnkey self-service solution, including sandbox testing and a simple setup, for providing robust and secure payment processing for your Commerce websites.
+
+## Payment services checkout workflow
+
+![Payment Services sequence diagram](../../_images/graphql/payment-services.svg)
+
+These steps describe the additional flow of calls required to complete a typical [Checkout](../tutorials/checkout/index.md) authorization with the `Payment Services` functionality enabled.
+
+1. Fetch the payment configuration with `GetPaymentConfig` in order to render payment methods, or PayPal components.
```

Comment:
I think the mutation names are case sensitive, so it should be `getPaymentConfig` with a lowercase G

Reply:
corrected it, thanks!